code/modules/modular_computers/hardware/battery_module.dm

```diff
@@ -15,15 +15,15 @@
 	desc = "An advanced power cell, often used in most laptops. It is too large to be fitted into smaller devices. It's rating is 1100."
 	icon_state = "battery_advanced"
 	origin_tech = list(TECH_POWER = 2, TECH_ENGINEERING = 2)
-	hardware_size = 3
+	hardware_size = 2
```

Comment:
Hardware size larger than device's size is actually preventing that hardware to be installed at all, that means if research has unlocked it and made it, then it still won't let it to be installed as "This component is too big for laptop".
I found it depressing that description says `...often used in most laptops.` and it can't fit in laptops.
But I think if that's what you want, then I can dail back some hardware sizes so they will be closer to closer to line when it wouldn't make sense.
Currently, only cheap options for laptops are available.

Reply:
Yes, thats the point, laptops are intentionally limited until we have figured out what we want to do with meta research.
Unlocking options that usually cant be unlocked will be part of meta research.